docker-compose.yml

```diff
@@ -1,32 +1,39 @@
 version: "3.5"
 services:
-  user_app:
+  cdslab_cmodels_app:
     build: .
-    container_name: cdslab_models_app
+    container_name: cdslab_cmodels_app
     env_file: .env
     image: cdslab_user
     networks:
-      - cdslab_models
+      - cdslab_cmodels_net
+    depends_on:
+      - cdslab_cmodels_mongo
     ports:
       - 5000:5000
     volumes:
-    - ./src:/app/src
+    - cdslab_cmodels_app_data:/app/src
```

Comment:
Same Volume Fix, should be:

```
volumes:
    - ./src:/app/src
```

Reply:
@davisfelipe according to [Docker documentation ](https://docs.docker.com/storage/volumes/#choose-the--v-or---mount-flag) the definition of volume consists on three parts separated by colons: <optional_name>:<path_to_volume>:<volume_options>. Supposing we omit the <optional_name> I don't see the reason why we are specifying two paths... It's supposed to be one path and some options...

Reply:
Thanks for the explanation... Drop this documentation here just for reference... https://docs.docker.com/storage/bind-mounts/